pkg/tasks/envfile.go

```diff
@@ -0,0 +1,20 @@
+package tasks
+
+import (
+	"github.com/devbuddy/devbuddy/pkg/tasks/taskapi"
+)
+
+func init() {
+	taskapi.Register("envfile", "EnvFile", parseEnvfile)
+}
+
+func parseEnvfile(config *taskapi.TaskConfig, task *taskapi.Task) error {
+	warning := func(ctx *taskapi.Context) error {
+		ctx.UI.TaskWarning("the .env file does NOT override existing variables")
+		ctx.UI.TaskWarning("the .env file is NOT unloaded when leaving the project")
```

Comment:
both not overriding & not reseting .env configuration would be weird sometime if you have 2 projects with a `.env` file no ? I'm wondering if it make sense not overriding?

Reply:
Yeah... I wanted to start implementing it to explore the requirements for https://github.com/devbuddy/devbuddy/issues/302

I think this should stay WIP until #302 is resolved